### PR TITLE
feat: add hello-universe 1.1.3 pack DOC-1273

### PR DESCRIPTION
**Pull request by addetz**
This PR includes a new version of the Hello Universe pack in the Spectro Cloud community repo. This pack allows users to specify tolerations for the Hello Universe application's UI, API, and Postgres pods. It also bumps the version of the Hello Universe image to 1.1.3. 

[Hello Universe](https://github.com/spectrocloud/hello-universe) is a demo web application that helps users learn more about Palette and its features.

I have tested this pack in our Prod environment across multiple clouds.

### CODE REVIEW

File: packs/hello-universe-1.1.3/README.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+The parameters are applied using the `Equal` operator as demonstrated below:
+
+```yaml
+    tolerations:
+    - effect: {{ .Values.ui.tolerations.effect }}
+    key: {{ .Values.ui.tolerations.key }}
+    operator: Equal
+    value: {{ .Values.ui.tolerations.value }}
+```
+
```
Comment (karl-cardenas-coding) [inline packs/hello-universe-1.1.3/README.md:74]: Is there any chance we could make this section its own H3?
That way it gets a heading and the reader can scan for easily :)

**Review by karl-cardenas-coding — DISMISSED** [review]
Looks good

File: packs/hello-universe-1.1.3/README.md (hunk 1 above)
Comment (addetz) [reply packs/hello-universe-1.1.3/README.md:76]: Added!

**Review by karl-cardenas-coding — APPROVED** [review]

**Review by karl-cardenas-coding — APPROVED** [review]

**Review by caroldelwing — APPROVED** [review]
:rocket:

**Review by vishwanaths — APPROVED** [review]

**Pull request merged**